src/claims/claims.controller.ts

```diff
@@ -0,0 +1,70 @@
+import { Body, Controller, HttpStatus, Post } from '@nestjs/common';
+import { ApiOperation, ApiTags } from '@nestjs/swagger';
+
+import { ClaimsService } from '~/claims/claims.service';
+import { ModifyClaimsDto } from '~/claims/dto/modify-claims.dto';
+import { ApiTransactionFailedResponse, ApiTransactionResponse } from '~/common/decorators/swagger';
+import { handleServiceResult, TransactionResponseModel } from '~/common/utils';
+import { PolymeshLogger } from '~/logger/polymesh-logger.service';
+
+@ApiTags('claims')
+@Controller('claims')
+export class ClaimsController {
+  constructor(
+    private readonly claimsService: ClaimsService,
+    private readonly logger: PolymeshLogger
+  ) {
+    logger.setContext(ClaimsController.name);
+  }
+
+  @ApiOperation({
+    summary: 'Get all Claims targeting an Identity',
+    description: 'This endpoint will provide a list of all the Claims made about an Identity',
+  })
```

Comment:
Isn't this endpoint for creating a claim? This reads like its for `@Get`, not `@Post`